libexec/scoop-info.ps1

```diff
@@ -143,7 +143,7 @@ if ($status.installed) {
         # Old versions = app total - current version size
         $fileTotals += $fileTotals[0] - $currentTotal
 
-        $item.'Installed size' = "Current version: $(filesize $currentTotal)`nOld versions: $(filesize $fileTotals[3])`nPersisted data: $(filesize $fileTotals[1])`nCached downloads: $(filesize $fileTotals[2])`nTotal: $(filesize $totalSize)"
+        $item.'Installed size' = "Current version:   $(filesize $currentTotal)`nOld versions:      $(filesize $fileTotals[3])`nPersisted data:    $(filesize $fileTotals[1])`nCached downloads:  $(filesize $fileTotals[2])`nTotal:             $(filesize $totalSize)"
```

Comment:
Split this string into into multiple lines

Reply:
Good idea, done!